pkg/networks/usernet/client.go

```diff
@@ -95,14 +96,12 @@ func (c *Client) ResolveIPAddress(vmMacAddr string) (string, error) {
 }
 
 func (c *Client) Leases() (map[string]string, error) {
-	res, err := c.client.Get(fmt.Sprintf("%s%s", c.base, "/services/dhcp/leases"))
+	u := fmt.Sprintf("%s%s", c.base, "/services/dhcp/leases")
+	res, err := httpclientutil.Get(context.TODO(), c.client, u)
 	if err != nil {
 		return nil, err
 	}
 	defer res.Body.Close()
-	if res.StatusCode != http.StatusOK {
-		return nil, fmt.Errorf("unexpected status: %d", res.StatusCode)
-	}
```

Comment:
Why remove?

Reply:
`httpclientutil.Get` already checks for successful status code, see https://github.com/lima-vm/lima/blob/a00bd52f78f86fc611f7c2a098597ae2892762ba/pkg/httpclientutil/httpclientutil.go#L28